projf-makefiles/common.mk

```diff
@@ -0,0 +1,59 @@
+BOARD_BUILDDIR := ${BUILDDIR}/${SUBPROJECT}
```

Comment:
I believe that this makefile is pretty much equal to the common makefile for the other projects. I think this is due to choosing another strategy for choosing the build directory, but I think it is better to keep consistency.

Possibly I'd put the build directory in the same place where the example's makefile is, and recall the `projf-makefiles` into `projf-examples`.

This way we avoid adding yet another common makefile to maintain.

Reply:
I have removed the extra common makefile and now the build designs will just be located in the same folder as the design makefile.